backends/occa/ceed-occa.c

```diff
@@ -16,6 +16,45 @@
 #define CEED_DEBUG_COLOR 10
 #include "ceed-occa.h"
 
+// *****************************************************************************
+// * CeedDladdr_Occa for Apple and Linux
+// *****************************************************************************
+#if defined(__APPLE__) || defined(__linux__)
+static int CeedDladdr_Occa(Ceed ceed) {   
+  Dl_info info;
+  Ceed_Occa *data = ceed->data;
+  memset(&info,0,sizeof(info));
+  const int ierr = dladdr((void*)&CeedInit,&info);
+  dbg("[CeedDladdr_Occa] found %s", info.dli_fname);
+  if (ierr==0)
+    return CeedError(ceed, 1, "OCCA backend cannot fetch dladdr");
+  // libceed_dir setup & alloc in our data
+  const char *libceed_dir = (char*)info.dli_fname;
```

Comment:
Isn't it already `const char *`, so no cast is needed?

Reply:
done